.github/workflows/e2e.generic.push.branch1.default.slsa3.yml

```diff
@@ -0,0 +1,128 @@
+name: Generic push branch1 default SLSA2
```

Comment:
rename to SLSA3

Reply:
Yeah, I need to do it for the lot of them.